change/@fluentui-react-charting-53e6e8eb-6b93-49b1-8419-7701781c49a0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31469&commitId=8cb073334dabcc70fe31c5114159f3e8187f85d4&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-SankeyChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SankeyChart.Basic.default.chromium.png|683| Changed|
|react-charting-SankeyChart.PlaceHolder.default.chromium.png|234| Changed|
|react-charting-SankeyChart.PlaceHolder - RTL.default.chromium.png|234| Changed|
|react-charting-SankeyChart.Basic - Dark Mode.default.chromium.png|467| Changed|
|react-charting-SankeyChart.Basic - RTL.default.chromium.png|682| Changed|
|react-charting-SankeyChart.PlaceHolder - Dark Mode.default.chromium.png|191| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>